deploy/lib/buildAndPushContainers.js

```diff
@@ -5,15 +5,37 @@ const tar = require('tar-fs');
 
 const docker = new Docker();
 
-const promisifyStream = (stream, verbose) => new Promise((resolve, reject) => {
-  stream.on('data', (data) => {
-    if (verbose) {
-      console.log(data.toString().replace('\n', ''));
-    }
+function extractStreamContents(stream, verbose) {
+  return new Promise((resolve, reject) => {
+    const streamContent = [];
+
+    stream.on('data', (data) => {
+      const streamData = data.toString().replace('\n', '')
+      streamContent.push(streamData);
+
+      if (verbose) {
+        console.log(streamData);
+      }
+    });
+
+    stream.on('end', () => {
+      resolve(streamContent);
+    });
+    stream.on('error', reject);
   });
-  stream.on('end', resolve);
-  stream.on('error', reject);
-});
+}
+
+function findErrorInBuildOutput(buildOutput) {
+  for (let i = 0; i < buildOutput.length; i++) {
```

Comment:
why not use `forEach` ?

Reply:
We can't return inside a `forEach`, but I've changed to `for (const ... of ...)`.